src/components/reward-claimed-toast/RewardClaimedToast.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useContext, useEffect } from 'react'
+
+import { useDispatch, useSelector } from 'react-redux'
+
+import { ReactComponent as IconCaretRight } from 'assets/img/iconCaretRight.svg'
+import { getShowRewardClaimedToast } from 'common/store/pages/audio-rewards/selectors'
+import { hideRewardClaimedToast } from 'common/store/pages/audio-rewards/slice'
+import { ToastContext } from 'components/toast/ToastContext'
+import ToastLinkContent from 'components/toast/mobile/ToastLinkContent'
+import { getLocationPathname } from 'store/routing/selectors'
+import { CLAIM_REWARD_TOAST_TIMEOUT_MILLIS } from 'utils/constants'
+import { AUDIO_PAGE } from 'utils/route'
+
+import styles from './RewardClaimedToast.module.css'
+
+const messages = {
+  challengeCompleted: 'You’ve Completed an $AUDIO Rewards Challenge!',
+  seeMore: 'See more'
+}
+
+export const RewardClaimedToast = () => {
+  const { toast } = useContext(ToastContext)
+  const dispatch = useDispatch()
+  const showToast = useSelector(getShowRewardClaimedToast)
+  const pathname = useSelector(getLocationPathname)
+
+  useEffect(() => {
+    if (showToast) {
+      const toastContent = (
+        <div className={styles.rewardClaimedToast}>
+          <span className={styles.rewardClaimedToastIcon}>
+            <i className='emoji face-with-party-horn-and-party-hat' />
+          </span>
+          &nbsp;&nbsp;
```

Comment:
can we use css instead of the &nbsp;?

Reply:
What do you mean? to show/hide the right message via css?

Reply:
i just mean use margins instead of &nbsp;

Reply:
omg `&nbsp;` github escapes that out, that's why my first post was confusing

Reply:
AHH that makes so much more sense haha